reports/Illinois.md

```diff
@@ -74,16 +74,20 @@ Victim states that officers kneeled on her neck, that shards of glass from her b
 
 After release (with neither charges nor explanation), the victim attempted to file a complaint but was told the complaint line was unnaccesible.
 
+This was reported to have happened at "2600 N. Narragansett Ave.", but 2700 N seems slightly more accurate. Google Streetview at that address shows the same "Show Carnival" store seen in the video.
```

Comment:
How far apart are 2600 and 2700 N. Narragansett?

Reply:
Couple hundred metres maybe

Reply:
Okay seems like a small enough difference to not really matter - tbh I think if you have the name of the shopping complex, that might be more informative?